vizier/backend/src/info/vizierdb/commands/plot/PlotUtils.scala

```diff
@@ -234,25 +330,38 @@ object PlotUtils
     ) =
       series.map { data =>
         val name = seriesName(data)
+        val encoding = VegaMarkEncoding(
+          x = Some(VegaValueReference.Field(data.x).scale("x")),
+          y = Some(VegaValueReference.Field(data.y).scale("y")),
+          stroke = Some(VegaValueReference.Literal(JsString(name)).scale("color")),
+          fill = 
+            if(!fill){ None }
+            else { Some(VegaValueReference.Literal(JsString(name)).scale("color")) },
+          tooltip = 
+            if(!tooltip){ None }
+            else { Some(VegaValueReference.Signal("datum")) },
+          opacity = 
+            if(opacity >= 1.0){ None }
+            else { Some(opacity) }
+        )
+
+        val updatedEncoding = 
+          if(markType == VegaMarkType.Rect) {
+            encoding.copy(
+              // y = Some(VegaValueReference.Field("sum(" + data.y + ")").scale("y")),
+              width = Some(VegaValueReference.ScaleBandRef("x", band = Some(1))),
+              y2 = Some(VegaValueReference.ScaleTransform("y", VegaValueReference.Literal(JsNumber(0))))
+            )
+          } else {
+            encoding
+          }
+
```

Comment:
For the pattern you see here: (i) Generate a variable and (ii) optionally apply specific modification steps, my preference is to use a `var`.  Creating a separate variable for each encoding step risks control-flow bugs when someone adds a step and forgets to update a variable name somewhere.

Suggested restructure:
```
  var encoding = ...

  if(markType == VegaMarkType.Rect) { 
    encoding = 
      ...
  }
```

`val` is generally preferred to `var` for a number of reasons, but for this *specific* pattern, I've found `var` to be less prone to bugs.